### PR TITLE
Use console_bridge_vendor dependency wrapper

### PR DESCRIPTION
**Pull request by henningkayser**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(Boost REQUIRED)
+find_package(console_bridge_vendor REQUIRED)
 find_package(console_bridge REQUIRED)
```
Comment (rhaschke) [inline CMakeLists.txt]: Do you need both, `console_bridge_vendor` and `console_bridge`?

**Review by rhaschke — COMMENTED** [review]
Looks good. One open question though.

File: CMakeLists.txt (hunk 1 above)
Comment (henningkayser) [reply CMakeLists.txt]: Hm, we can probably omit it, it's not exporting any targets. I just implemented this in accordance to other packages.

File: package.xml [hunk 2]
```diff
@@ -16,15 +16,15 @@
...
-  <build_depend>libconsole-bridge-dev</build_depend>
+  <build_depend>console_bridge_vendor</build_depend>
   <build_depend>urdfdom_headers</build_depend>
   <build_depend>urdfdom_py</build_depend>
   <build_depend>tinyxml2_vendor</build_depend>
   <build_depend>urdf</build_depend>
 
   <exec_depend>urdf</exec_depend>
   <exec_depend>boost</exec_depend>
-  <exec_depend>libconsole-bridge-dev</exec_depend>
```
Comment (jacobperron) [inline package.xml:24]: I think you can depend on both `libconsole-bridge-dev` **and** `console_bridge_vendor`.
This way `rosdep` will install the upstream version of `console_bridge` in Debian, and if (at some point in the future) it meets the minimum version requirement of `console_bridge_vendor`, we can avoid building it from source.

**Comment by seanyen** [comment]
+1. Moving toward to `console_bridge_vendor` would make the package more friendly for other platforms (E.g., Windows. 😉)

**Review by tylerjw — APPROVED** [review]

**Pull request merged**